python/ray/tune/schedulers/pb2.py

```diff
@@ -122,26 +120,20 @@ def _select_config(
 
     fixed = normalize(newpoint, oldpoints)
 
-    kernel = TV_SquaredExp(
-        input_dim=X.shape[1], variance=1.0, lengthscale=1.0, epsilon=0.1
-    )
+    kernel = TV_SquaredExp(variance=1.0, lengthscale=1.0, epsilon=0.1)
 
     try:
-        m = GPy.models.GPRegression(X, y, kernel)
-    except np.linalg.LinAlgError:
-        # add diagonal ** we would ideally make this something more robust...
-        X += np.eye(X.shape[0]) * 1e-3
-        m = GPy.models.GPRegression(X, y, kernel)
-
-    try:
-        m.optimize()
+        m = GaussianProcessRegressor(
```

Comment:
Is `m = GPy.models.GPRegression(X, y, kernel)` equivalent to 

```
m = GaussianProcessRegressor(
    kernel=kernel, optimizer="fmin_l_bfgs_b", alpha=1e-10
)
m.fit(X, y)
```

? Might be worth doublechecking if this matters.